File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L6/08/Kasus4L6.kt
package IR

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus4L6 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        ShowTable(10)
    }

    private fun ShowTable(jumlahData: Int) {
        for (i in jumlahData downTo 1) {
            //Menampilkan hasil
            val temp = jumlahData + 1 - i
            println(temp.toString() + "\t\t" + ConvertMilesToKilo(temp))
        }
    }

    private fun ConvertMilesToKilo(miles: Int): Double {
        return miles * 1.609
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L6/08/Kasus4L6.ts
/**
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
function showTable(jumlahData: number): void {
    console.log("Miles\t\tKilometers");
    console.log("-------------------------------");
    for (let i = jumlahData; i >= 1; i--) {
        const temp = jumlahData + 1 - i;
        console.log(`${temp}\t\t${convertMilesToKilo(temp)}`);
    }
}

function convertMilesToKilo(miles: number): number {
    return miles * 1.609;
}

function main(): void {
    showTable(10);
}

main();

